osquery/logger/plugins/aws_util.cpp

```diff
@@ -356,4 +357,43 @@ Status getAWSRegion(std::string& region, bool sts) {
   VLOG(1) << "Using default AWS region: " << region;
   return Status(0);
 }
+
+Status appendLogTypeToJson(const std::string& log_type,
```

Comment:
Alternative, we _could_ "assume" that the `log` in `log_data` is already in proper JSON string format and append our `log_type` key inside it... but it doesn't feel proper.